include/odp/api/spec/pool.h

```diff
@@ -93,10 +93,17 @@ typedef struct odp_pool_capability_t {
 		/** Minimum packet level headroom length in bytes
 		 *
 		 * The minimum number of headroom bytes that newly created
-		 * packets have by default. The default apply to both ODP
+		 * packets can be configured. This value applies to both ODP
 		 * packet input and user allocated packets.*/
 		uint32_t min_headroom;
```

Comment:
Just to elaborate, it's always valid for an application to request 0 headroom. Whether that results in packets having 0 headroom is up to the implementation since implementations are free to round up application requests to something larger. So I agree with Petri that this particular wording change probably isn't needed. It simply says that in the absence of any specific requests from the application, this is the minimum amount of headroom that applications can expect to find in every newly received or allocated packet.

Reply:
In that case the naming "default_headroom" makes more sense than "min_headroom". Maybe we can leave it as "min_headroom" in this release.

Reply:
We should change this field since that's not required for the new feature to be functional.